chronicles.nim

```diff
@@ -197,7 +201,12 @@ macro logIMPL(Stream: typed,
         enabledTopicsMatch = true
       elif t in requiredTopics:
         dec requiredTopicsCount
-
+  
+  # Handling file name and line numbers on/off (lineNumbersEnabled) for particular log statements
+  if finalBindings.hasKey("chroniclesLineNumbers"):
+    lineNumbersSpec = $finalBindings["chroniclesLineNumbers"] == "true"
```

Comment:
This approach results in  "true" / "false" and true/false as allowed values. Perhaps this is intended, but originally true/false was requested in the issue. Allowing also "true" / "false" could give issues in the future if this is not stated in the documentation for example. So my suggestion would be to either change this or change the spec (and future documentation).

There is also no checking done if the value is anything else than these values, it will allow any other string and return it as false. I would rather like to see an error if I typed "truee".